src/legacy/ui/public/agg_types/controls/field.test.tsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mount, shallow, ReactWrapper } from 'enzyme';
+import { EuiComboBoxProps, EuiComboBox } from '@elastic/eui';
+import { Field } from '../../index_patterns';
+import { ComboBoxGroupedOptions, SubAggParamsProp } from '../../vis/editors/default';
+import { FieldParamEditor, FieldParamEditorProps } from './field';
+import { AggConfig, VisState } from '../../vis';
+
+function callComboBoxOnChange(comp: ReactWrapper, value: any = []) {
+  const comboBoxProps: EuiComboBoxProps<string> = comp.find(EuiComboBox).props();
+  if (comboBoxProps.onChange) {
+    comboBoxProps.onChange(value);
+  }
+}
+
+describe('FieldParamEditor component', () => {
+  let setValue: jest.Mock;
+  let setValidity: jest.Mock;
+  let setTouched: jest.Mock;
+  let onChange: jest.Mock;
+  let defaultProps: FieldParamEditorProps;
+  let indexedFields: ComboBoxGroupedOptions<Field>;
+  let field: Field;
+  let option: {
+    label: string;
+    target: Field;
+  };
+
+  beforeEach(() => {
+    setValue = jest.fn();
+    setValidity = jest.fn();
+    setTouched = jest.fn();
+    onChange = jest.fn();
+
+    field = { displayName: 'bytes' } as Field;
+    option = { label: 'bytes', target: field };
+    indexedFields = [
+      {
+        label: 'Field',
+        options: [option],
+      },
+    ];
+
+    defaultProps = {
+      agg: {},
+      aggParam: {
+        name: 'field',
+        type: 'field',
+        editorComponent: null as any,
```

Comment:
nit: `() => null` is just as long and works without `any`

Reply:
Thanks, done.